dart/lib/src/event_processor/enricher/io_enricher_event_processor.dart

```diff
@@ -40,6 +40,11 @@ class IoEnricherEventProcessor implements EnricherEventProcessor {
     );
 
     contexts['dart_context'] = _getDartContext();
+    contexts['process_info'] = <String, dynamic>{
+      'currentResidentSetSize':
+          _bytesToHumanReadableFileSize(ProcessInfo.currentRss),
+      'maxResidentSetSize': _bytesToHumanReadableFileSize(ProcessInfo.maxRss),
```

Comment:
are there any alternatives for web?

Reply:
this is based on this https://developer.mozilla.org/en-US/docs/Web/API/Performance/memory

it's deprecated and not all browsers support it, dunno if it's worth the effort to add this if we have to remove it again at some point

Reply:
Yeah, i removed it after reading the docs and added some info.